.github/BOTMETA.yml

```
@@ -570,6 +570,9 @@ files:
   $module_utils/docker_common.py:
     maintainers: $team_docker
     supershipit: felixfontein
+  $module_utils/docker_swarm.py:
+    maintainers: $team_docker
```

In case you want to be informed as well for PRs and issues involving module_utils/docker_swarm.py, you can also add yourself:
```suggestion
    maintainers: $team_docker WojciechowskiPiotr
```
(Alternatively, you can add yourself to the $team_docker definition at the bottom of the file if you are interested on other docker_* modules as well. If you don't do that, you're only informed when one of the modules you are listed as author for is affected.)

---

I'll add myself to $team_docker as I don't mind participating actively